packages/website/versioned_docs/version-legacy/dropdown.md

```diff
@@ -23,7 +23,7 @@ Once your Algolia DocSearch index is ready, set up, and filled with the right da
     appId: '<YOUR_APP_ID>',
     // Replace inputSelector with a CSS selector
     // matching your search input
-    inputSelector: '<YOUR_CSS_SELECTOR>',
```

Comment:
this is the documentation for docsearch v2, when it was still named `inputSelector`